scripts/tf_cnn_benchmarks/models/resnet_model.py

```diff
@@ -77,6 +77,48 @@ def bottleneck_block_v1(cnn, depth, depth_bottleneck, stride):
     cnn.top_layer = output
     cnn.top_size = depth
 
+def bottleneck_block_v15(cnn, depth, depth_bottleneck, stride):
```

Comment:
IMO v15 should be v1_5. That way, no one thinks this is the 15th resnet version :)

Reply:
I think XQ agrees.  I so as well.

Reply:
Fixed.